packages/nimble-components/src/table-column/number-text/models/decimal-formatter.ts

```diff
@@ -13,12 +14,19 @@ export class DecimalFormatter extends NumberFormatter {
             minimumFractionDigits: decimalsToDisplay,
             useGrouping: true
         });
+        this.tenPowDecimalDigits = 10 ** decimalsToDisplay;
     }
 
     protected format(number: number): string {
         // The NumberFormat option of `signDisplay: "negative"` is not supported in all browsers nimble supports.
```

Comment:
I should have asked for this when the comment was first added, but can you update the comment to explain why we don't want to render -0? Was it because grouping doesn't distinguish between +0 and -0?

Reply:
We did this with our other two formatters as well (`round-to-integer` and `default`). The thought is that for most cases, people don't think about -0 being different than +0, so we didn't think it made sense for any of our built-in formatters to display -0.

Grouping, however, is an interesting question. Grouping will always be based on the raw value, but it will be rendered based on the formatter. For example, 1.23 and 1.22 with 1 decimal will both render as 1.2, but they won't group together because their values are different. That is a separate concern than -0 though.

Reply:
Yea we might want to make a more descriptive usage note for the grouping docs about this that can be linked to from the number column formatting docs. Think we would need to create a docs page for grouping to do that. Think that would be separate from this PR though.